.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target: [
+            aarch64-unknown-linux-gnu,
+            x86_64-linux-android,
+            i686-linux-android,
+            armv7-linux-androideabi,
+            aarch64-linux-android
+          ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: "Cache"
```

Comment:
This doesn't seem to be super effective in reducing run time of the workflow, so could probably be removed.